test/integration/commands/help-test.ts

```diff
@@ -3,10 +3,12 @@ import HelpCommand from "../../../src/commands/help";
 import { CommandArgs } from "../../../src/util/commandline";
 import { expect } from "chai";
 
-describe("Validating help command", () => {
+describe("Validating help command", function () {
 
   let stubbedConsole: Sinon.SinonStub;
 
+  this.timeout(5000);
```

Comment:
I slipped this in -- maybe it belongs to a separate PR?

Reply:
Fine with me